base/bin/migration.mustache

```diff
@@ -12,12 +12,13 @@ const migrationTestTemplate = fs.readFileSync('./bin/migration_template_test.mus
 const migrationFile = mustache.render(migrationTemplate, opts);
 const migrationTest = mustache.render(migrationTestTemplate, opts);
 
-try {
-  fs.mkdirSync('./migrations/');
-  fs.mkdirSync('./test/migrations/');
-} catch (e) {
-  if (e.code !== 'EEXIST') throw e;
-}
+['./migrations/', './test/migrations'].forEach(p => {
```

Comment:
Oh, yeah. It doesn't really matter either way. Would be nice to have it symmetrical though.